src/components/Indicator.js

```diff
@@ -56,7 +59,8 @@ const defaultProps = {
     allPolicyMembers: {},
     policies: {},
     bankAccountList: {},
-    cardList: {},
+    cardList: null,
```

Comment:
Do we treat null and empty object differently in functions that use cardList/fundList?

Reply:
I presume this change was made because `null` is falsy and an empty object isn't. If we defaulted to an empty object for `fundList` then the short circuit on line 75 would lead to `paymentCardList` being an empty object instead of falling back on the `cardList`.